traffic_monitor/experimental/traffic_monitor/srvhttp/srvhttp.go

```diff
@@ -134,22 +136,29 @@ func ParametersStr(params url.Values) string {
 	return pp
 }
 
+//CommonAPIDataDataFormat is a common Date format for the API
 const CommonAPIDataDateFormat = "Mon Jan 02 15:04:05 UTC 2006"
 
 // DateStr returns the given time in the format expected by Traffic Monitor 1.0 API users
 func DateStr(t time.Time) string {
 	return t.UTC().Format(CommonAPIDataDateFormat)
 }
 
-func (s Server) handleRootFunc() (http.HandlerFunc, error) {
-	return s.handleFile("index.html")
+func (s *Server) handleRootFunc(staticFileDir string) (http.HandlerFunc, error) {
+	if strings.HasSuffix(staticFileDir, "/") {
+		return s.handleFile(staticFileDir + "index.html")
+	}
+	return s.handleFile(staticFileDir + "/index.html")
 }
 
-func (s Server) handleSortableFunc() (http.HandlerFunc, error) {
-	return s.handleFile("sorttable.js")
+func (s *Server) handleSortableFunc(staticFileDir string) (http.HandlerFunc, error) {
+	if strings.HasSuffix(staticFileDir, "/") {
+		return s.handleFile(staticFileDir + "sorttable.js")
+	}
+	return s.handleFile(staticFileDir + "/sorttable.js")
```

Comment:
Not a fan of duplicating this line. POSIX guarantees ignoring duplicated slashes, so could we just remove the `if` and always `return s.handleFile(staticFileDir + "/sorttable.js")`? So it could be "path/to//sortable.js", but that's ok?